src/containers/MainContainer.jsx

```diff
@@ -1,6 +1,23 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
+import { increment } from '../actions/actionCreators';
 
-const MainContainer = () => <div>This is MainContainer</div>;
+//retrieve state from store 
+const mapStatetoProps = store => ({
+  count: store.games.count,
+});
 
-export default MainContainer;
+const mapDispatchToProps = dispatch => ({
+  increment: () => { dispatch(increment()) }
+})
+
+const MainContainer = ({ count, increment }) => (
+  < div >
```

Comment:
this is good for testing but lets plan to remove this since it is not apart of the apps functionality